extensions/omni/docs/intro.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Managing & Developing Extensions
+---
+
+# Intro to omni
+
+`omni` is a Postgres shared library (and also available as an extension) that helps managing and developing Postgres
+extensions easier. Think of it as an extension for extensions !
+
+# Why?
+
+The ecosystem of Postgres extensions is expanding at a fast pace, but the supporting tools and infrastructure are
+lagging.
+For context, an extension that allocates shared memory or initiates background workers necessitates a Postgres server
+restart.
+Another challenge is setting up multiple hooks of the same type and properly accessing them in the right order.
+
+# Features
+
+* Hooks initialization
+* Allocate/Deallocate shared memory
+* Managing background workers
+* Upgrading extensions
+* Setting custom GUC
+
+# Installing omni
+
+## Download omni
+export omni_latest_version=$(curl -s https://index.omnigres.com/16/Release/ubuntu-x86-64/index.json | jq -r '.extensions.omni | keys | sort | .[-1]')
+
+`curl -s https://raw.githubusercontent.com/omnigres/omnigres/master/download-omnigres-extension.sh | bash -s install omni $omni_latest_version`
+
+Option: Refer [omni_manifest](https://docs.omnigres.org/omni_manifest/usage/#install) for more on installing extensions
```

Comment:
this can be a relative link without the domain so that it remains correct even if the domain changes